### PR TITLE
Display playlists and add to playlist

### PR DESCRIPTION
**Pull request by LeszekKrz**

### CODE REVIEW

File: frontend/src/app/pages/choose-playlist/choose-playlist.component.spec.ts
Comment (Marvvvin5) [inline frontend/src/app/pages/choose-playlist/choose-playlist.component.spec.ts:1]: if its empty just delete it

File: frontend/src/app/pages/home/home.component.html [hunk 1]
```diff
@@ -43,3 +43,5 @@
 </div>
 
 </p-card>
+
+<p-card></p-card>
```
Comment (Marvvvin5) [inline frontend/src/app/pages/home/home.component.html]: what is it?

File: frontend/src/environments/environment.development.ts [hunk 2]
```diff
@@ -4,8 +4,8 @@
 
 export const environment = {
     production: false,
-    webApiUrl: 'https://localhost:7133',
-    webUrl: 'https://localhost:7133',
+    webApiUrl: 'https://wetubeback.azurewebsites.net/',
```
Comment (Marvvvin5) [inline frontend/src/environments/environment.development.ts]: probably we should stick with localhost. You can keep the other one commented

Comment (KarolDenst) [inline frontend/src/environments/environment.development.ts]: are you sure about this change?

File: frontend/src/app/pages/choose-playlist/choose-playlist.component.spec.ts
Comment (LeszekKrz) [reply frontend/src/app/pages/choose-playlist/choose-playlist.component.spec.ts:1]: Fixed

File: frontend/src/app/pages/home/home.component.html (hunk 1 above)
Comment (LeszekKrz) [reply frontend/src/app/pages/home/home.component.html]: Must have been added accidentally, fixed

File: frontend/src/environments/environment.development.ts (hunk 2 above)
Comment (LeszekKrz) [reply frontend/src/environments/environment.development.ts]: Done

**Review by KarolDenst — APPROVED** [review]

**Pull request merged**